src/main/scala/devices/gpio/GPIO.scala

```diff
@@ -135,30 +135,29 @@ trait HasGPIOModuleContents extends MultiIOModule with HasRegMap {
   val rise = ~valueReg & inSyncReg;
   val fall = valueReg & ~inSyncReg;
 
-  val iofEnFields =  if (c.includeIOF) (Seq(RegField.rwReg(c.width, iofEnReg.io))) else (Seq(RegField(c.width)))
-  val iofSelFields = if (c.includeIOF) (Seq(RegField(c.width, iofSelReg))) else (Seq(RegField(c.width)))
+  val iofEnFields =  if (c.includeIOF) (Seq(RegField.rwReg(c.width, iofEnReg.io, Some(RegFieldDesc("iof_en","HW I/O functon enable", reset=Some(0)))))) else (Seq(RegField(c.width)))
```

Comment:
these lines are so long it's hard to actually review the PR... can we break at the descriptions comma? Or use `DescribedReg`?